buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -677,18 +677,18 @@ def setup_pull_request() {
        Jenkins test extended <platform>
        It was tested*/
 
-    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
-    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
-        def comment = COMMENT.toLowerCase().tokenize(' ')
-        if (("${comment[0]}" == "jenkins") && (("${comment[1]}" == "compile") || ("${comment[1]}" == "test"))) {
-            setup_pull_request_single_comment(comment)
+    def parsedByNewlineComment = ghprbCommentBody.split(/\\r?\\n/)
+    for (comment in parsedByNewlineComment) {
+        def lowerCaseComment = comment.toLowerCase().tokenize(' ')
+        if (("${lowerCaseComment[0]}" == "jenkins") && (("${lowerCaseComment[1]}" == "compile") || ("${lowerCaseComment[1]}" == "test"))) {
```

Comment:
maybe simplify 
`(("${lowerCaseComment[1]}" == "compile") || ("${lowerCaseComment[1]}" == "test"))`
to 
`(["compile", "test"].contains(lowerCaseComment[1])`

Reply:
changed